test/unit/alignment/aligned_sequence_test_template.hpp

```diff
@@ -36,16 +36,16 @@ TYPED_TEST_P(aligned_sequence, fulfills_concept)
 
 TYPED_TEST_P(aligned_sequence, assign_unaligned_sequence)
 {
```

Comment:
The changes in the file are not related to `gap_decorator::unaligned_seq_type` - just revert them